simple_network_sim/sampleUseOfModel.py

```diff
@@ -178,31 +185,9 @@ def build_args(argv):
         description="Uses the deterministic network of populations model to simulation the disease progression",
     )
     parser.add_argument(
-        "--compartment-transition",
-        default=sampledir / "compartmentTransitionByAge.csv",
-        help="Epidemiological rate parameters for movement within the compartmental model.",
-    )
-    parser.add_argument(
-        "--population",
-        default=sampledir / "sample_hb2019_pop_est_2018_row_based.csv",
-        type=Path,
-        help="This file contains age-and-sex-stratified population numbers by geographic unit.",
-    )
-    parser.add_argument(
-        "--commutes",
-        default=sampledir / "sample_scotHB_commute_moves_wu01.csv",
-        type=Path,
-        help="This contains origin-destination flow data during peacetime for health boards",
```

Comment:
Now that we don't have those parameters anymore, how do we select different data files to use? Is it through that data config file?

Reply:
Yes, it will be.